feegrant/index.md

```diff
@@ -0,0 +1,239 @@
+---
+parent:
+  title: Understanding the Fee Grant Module
+order: 0
+description: Use the Cosmos SDK feegrant module to grant the use of tokens to pay for fees from one account (the granter) to another account (the grantee).
+---
+
+# Feegrant Module
+
+The [feegrant](https://docs.cosmos.network/v0.44/modules/feegrant/) enables a user / contract / module (the granter) to pay the fees for someone else (the grantee) when they want to broadcast a transaction on the blockchain. A key feature of feegrant is that the granter retains full access to their tokens, and is able to revoke the allowance at any time.
+
+## Use Feegrant to Grant an Allowance
+
+An often discussed use case for the feegrant module is that blockchains or smart contracts can allow new users to start interacting with the chain or contract without having to acquire tokens first, greatly improving the onboarding experience.
+
+There are two [fee allowance types](https://docs.cosmos.network/v0.44/modules/feegrant/01_concepts.html#fee-allowance-types) that the feegrant module implements: `BasicAllowance` and `PeriodicAllowance`, the latter allows you to setup a periodic allowance to another user. In this tutorial, you will set up two tokens in your blockchain: one default token called `stake`, which will be used for fees, and another called `kudos`, for sending to your friends.
+
+You will learn how to spin up a single node network using the simulation application in Cosmos SDK (`simapp`) and set up Alice to be a validator. Then you'll have Bob send `kudos` tokens to Carol, while having zero `stake` to pay for fees. The latter will be possible because you'll let Carol grant a `BasicAllowance` to Bob.
```

Comment:
```suggestion
- You will learn how to spin up a single node network using the simulation application in Cosmos SDK (`simapp`).
- Set Alice up to be a validator.
- Carol is the granter and grants a `BasicAllowance` to Bob.
- Bob is the grantee who receives a `BasicAllowance` that allows Bob to send `kudos` tokens to Carol, even though Bob has zero `stake` to pay for fees. 
```

Reply:
I suggest that we avoid "the latter" and other positional references, let's r/w for clarity and simple word flow 

This tutorial overview is a good time to identify the role of each user

Reply:
This is much clearer, thanks! Changing the order of Bob and Carol according to @xavierlepretre feedback which mentioned the standard of introducing users alphabetically. 

```suggestion
- You will learn how to spin up a single node network using the simulation application in Cosmos SDK (`simapp`).
- Set Alice up to be a validator.
- Bob is the grantee who receives a `BasicAllowance` that allows Bob to send `kudos` tokens to Carol, even though Bob has zero `stake` to pay for fees. 
- Carol is the granter and grants a `BasicAllowance` to Bob.
```